qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -17,13 +17,40 @@
             no RHEL.6 RHEL.7
             no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
             required_qemu = [8.2.0,)
+            check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
     variants:
         - default:
             no s390x
         - siblings_core:
-            only s390x
-            RHEL.8:
-                check_siblings_cmd = "awk -F'-' '{print $2 - $1 + 1}' /sys/devices/system/cpu/cpu0/topology/core_siblings_list"
-            RHEL.9:
-                check_siblings_cmd = 'cat /proc/cpuinfo |grep siblings |uniq |cut -d ":" -f 2'
-            check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
+            no aarch64
+            s390x:
+                RHEL.8:
+                    check_siblings_cmd = "awk -F'-' '{print $2 - $1 + 1}' /sys/devices/system/cpu/cpu0/topology/core_siblings_list"
+        - check_core_per_sockets:
+            type = cpu_topology_details_test
+            start_vm = yes
+            check_core_per_socket_cmd = "lscpu --extended | awk 'NR>1 {print $3}'"
+            check_sockets_cmd = "lscpu | grep 'Socket(s)' |uniq |cut -d ":" -f 2"
```

Comment:
Even this works, but for human readable, it becomes to `"lscpu | grep 'Socket(s)' |uniq |cut -d "` + `" -f 2"`, suggest removing the outermost "

Reply:
updated